tests/functional/modules/test_zos_backup_restore.py

```diff
@@ -444,8 +440,6 @@ def test_backup_and_restore_of_multiple_data_sets(ansible_zos_module, data_set_i
             hlq=NEW_HLQ,
         )
         assert_module_did_not_fail(results)
-        assert_data_set_exists(hosts, DATA_SET_RESTORE_LOCATION)
```

Comment:
Also why is this assertion removed? I understand the current test case intents to verify that the data set exists after restore